Tools/WinMLRunner/BindingUtilities.h

```diff
@@ -417,4 +417,96 @@ namespace BindingUtilities
             std::cout << std::endl;
         }
     }
+
+    void SaveEvaluationResults(const LearningModel &model, const CommandLineArgs &args, const IMapView<hstring, winrt::Windows::Foundation::IInspectable>& results, OutputHelper &output, uint32_t iterationNum)
+    {
+        std::string iterationResult;
+        size_t hash = 0;
+
+        for (auto&& desc : model.OutputFeatures())
+        {
+            if (desc.Kind() == LearningModelFeatureKind::Tensor)
+            {
+                std::wstring name(desc.Name());
+                std::string tempName;
+                for (int i = 0; i < name.size(); i++)
+                {
+                    if (isalnum(name[i]))
+                    {
+                        tempName += name[i];
+                    }
+                }
+                TensorFeatureDescriptor tensorDescriptor = desc.as<TensorFeatureDescriptor>();
+                TensorKind tensorKind = tensorDescriptor.TensorKind();
+
+                switch (tensorKind)
+                {
+                case TensorKind::String:
+                {
+                    com_ptr<ITensorNative> itn = results.Lookup(desc.Name()).as<ITensorNative>();
+                    std::string* Tensor;
+                    uint32_t uCapacity;
+                    HRESULT(itn->GetBuffer(reinterpret_cast<BYTE**>(&Tensor), &uCapacity));
```

Comment:
This is not supported for TensorString types. The GetBuffer method will return ERROR_INVALID_FUNCTION here.